<p>Locators based on css should not be tied to page layout. Page layout can be changed frequently.
  Css locators that contains a "<code> </code>" (space) defines it to be tied to page layout.</p>
<p>Good locators are unique, descriptive and unlikely to change.</p>
<h2>Noncompliant Code Example</h2>
<pre>
  By.cssSelector("#form div input"); // Noncompliant
</pre>
<h2>Compliant Solution</h2>
<pre>
  By.cssSelector("#submitButton");
</pre>
